recipes/cli11/all/test_package/CMakeLists.txt

```
@@ -1,11 +1,10 @@
 cmake_minimum_required(VERSION 3.1)
```

```suggestion
cmake_minimum_required(VERSION 3.8)
```

cxx_std_11 requires 3.8: https://cmake.org/cmake/help/latest/prop_gbl/CMAKE_CXX_KNOWN_FEATURES.html#high-level-meta-features-indicating-c-standard-support